src/app.js

```diff
@@ -1 +1,24 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+function createLogFile() {
+  const currentDate = new Date();
+  const hours = currentDate.getHours().toString().padStart(2, '0');
```

Comment:
You can move this logic `.toString().padStart(2, '0')` to a separate method to not repeat the same logic everywhere